docs/proposals/compute-resource-metrics-api.md

```diff
@@ -132,16 +122,12 @@ As well as the common query parameters:
 
 ### Rationale
 
-We are not adding new methods to pods and nodes, e.g.
-`/api/v1/namespaces/myns/pods/mypod/metrics`, for a number of reasons. For
-example, having a separate endpoint allows fetching all the pod metrics in a
-single request. The rate of change of the data is also too high to include in
-the pod resource.
+We are not adding new methods to pods and nodes, e.g. `/api/v1/namespaces/myns/pods/mypod/metrics`, for a number of reasons.
+For example, having a separate endpoint allows fetching all the pod metrics in a single request.
+The rate of change of the data is also too high to include in the pod resource.
+A separate endpoint helps in storing metrics data in a metrics friendly backend storage.
 
-In the future, if any uses cases are found that would benefit from RC,
-namespace or service aggregation, metrics at those levels could also be
-exposed taking advantage of the fact that Heapster already does aggregation
-and metrics for them.
+In the future, if any uses cases are found that would benefit from RC, namespace or service aggregation, metrics at those levels could also be exposed taking advantage of the fact that Heapster already does aggregation and metrics for them.
```

Comment:
It's not required in the near-term, but it would be great to have namespace aggregation if its not much additional effort.